zc_common/remote_resource/tests.py

```diff
@@ -127,24 +134,10 @@ def success_response_structure_test(self, response, status, relationship_keys=No
                     self.assertTrue(
                         all(key in relationship for key in ['data', 'links']))
 
-                    for relationship_data in self.convert_to_list(relationship['data']):
```

Comment:
Why are we removing these parts of the test? they seem to be testing very critical parts of the response. I would agree that the attributes that this method uses could be better documented, but lets refrain from removing the tests.